config/stage.json

```diff
@@ -46,6 +46,7 @@
 		"google-my-business": true,
 		"gutenboarding/mshot-preview": false,
 		"gutenboarding/style-preview-verticals": false,
+		"gutenboarding/edge-templates": true,
```

Comment:
Do we need to have Staging show the edge templates as well? Just thinking that for debugging something happening in Production, it might be easier for us if Staging is as close as possible to what the user is seeing in Production, and we could use horizon or wpcalypso for testing Edge templates?

Reply:
> Do we need to have Staging show the edge templates as well? 

No we absolutely don't. Thanks a lot for catching this. It was a shotgun approach. I'll fix now.